klippa/src/lib.rs

```diff
@@ -807,13 +818,14 @@ pub trait Subset {
 // A helper trait providing a 'subset' method for various subtables that have no associated tag
 pub(crate) trait SubsetTable<'a> {
     type ArgsForSubset: 'a;
+    type SubsetOutput: 'a;
     /// Subset this table and write a subset version of this table into serializer
     fn subset(
         &self,
         plan: &Plan,
```

Comment:
I'd just call this `Output`, we know it's part of the `SubsetTable` trait. (Looking back I would have also suggested `Args` instead of `ArgsForSubset` but maybe that name was about avoiding clashing with `FontRead`?